categories/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+class Category(models.Model):
+    name = models.CharField(max_length=50)
+
+    class Meta:
+        db_table = 'categories'
+
+class Sub_Category(models.Model):
```

Comment:
class 명은 아래와 같이 작성해주세요~ 모든 클래스에 적용해주세요!
```suggestion
class SubCategory(models.Model):
```